Status Bar & Layout Cleanup

jameskerr opened the pull request:
![05DqT3HnZR](https://user-images.githubusercontent.com/3460638/178615554-9de5cc62-066f-499c-abe1-4569981ebbb2.gif)

While I was doing this, I changed the AppWrapper class to use a css grid layout for the app. I added grid-areas to the main components of the page.

I also gutted the Pane class and renamed it "DraggablePane". 

I deleted some old css and component files.

mason-fish reviewed (commented):
I checked this out and the styling looks great to me. Couldn't find any issues with the new grid layout 👍 

I did find 2 other cases we should handle:
* If I issue a search, and then issue another before the first has finished. The status gets updated to "Completed" even though it is still fetching
* If I switch tabs after issuing a search, the completed status gets updated on the new current tab and the original tab remains in the same stale fetching status.

jameskerr commented:
@mason-fish you found a big bug! Results were always going to the "active" tab instead of the tab specified when the query began. I've fixed it in here.
<img width="405" alt="image" src="https://user-images.githubusercontent.com/3460638/179314175-aad715dc-77f1-46b8-ad3d-0dd8a9840bb8.png">

<img width="276" alt="image" src="https://user-images.githubusercontent.com/3460638/179314199-749d920f-ab87-4760-9083-b116e66958f0.png">

<img width="345" alt="image" src="https://user-images.githubusercontent.com/3460638/179314223-e1a0b2d5-e0a2-473d-b3ee-aa9f324ed5ce.png">

I've added the number of shapes in the data as well.

jameskerr commented on packages/zealot/src/query/result-stream.ts:
```diff
@@ -87,7 +87,7 @@ export class ResultStream {
         ) {
           this.status = "aborted"
-          resolve()
+          reject(e)
```
This fixes the first error you found. When aborting, it was still resulting in success. Not anymore.

jameskerr commented on src/js/state/Tabs/slice.ts:
```diff
@@ -85,7 +85,7 @@ const slice = createSlice({
   extraReducers: (builder) => {
     builder.addMatcher(isTabAction, (s, a) => {
-      const id = a.tabId || s.active
+      const id = a.tabId || a.payload?.tabId || s.active
```
This fixes the second problem you found. Where the results and the stats are updating the wrong tab.

mason-fish reviewed (approved):
Awesome!
